qiskit_experiments/analysis/curve_fit_analysis.py

```diff
@@ -0,0 +1,165 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Curve fitter analysis class
+"""
+
+from typing import Union, Optional, Callable, List, Tuple, Dict
+
+import numpy as np
+from scipy.optimize import curve_fit
```

Comment:
You'll need to add scipy to the requirements list, I think it only has numpy and qiskit-terra in there now.

Reply:
CI tests pass in #5, where `curve_fit` is also imported

Reply:
Well, actually my mistake, they don't pass, but for another reason (#16 and linter). So perhaps it just didn't get to the stage of checking this import.